plugins/plugin-docker/che-plugin-openshift-client/src/main/java/org/eclipse/che/plugin/openshift/client/OpenShiftConnector.java

```diff
@@ -1530,6 +1532,7 @@ private String waitAndRetrieveContainerID(String deploymentName) throws IOExcept
                 if (OPENSHIFT_POD_STATUS_RUNNING.equals(status)) {
                     String containerID = pod.getStatus().getContainerStatuses().get(0).getContainerID();
                     String normalizedID = KubernetesStringUtils.normalizeContainerID(containerID);
+                    pod.getSpec().setTerminationGracePeriodSeconds(OPENSHIFT_POD_TERMINATION_GRACE_PERIOD);
```

Comment:
Doing it this way will not update the actual Pod spec. I think we need to add this in `createOpenShiftDeployment()`

```
        PodSpec podSpec = new PodSpecBuilder()
                                 .withContainers(container)
                                 .withVolumes(getVolumesFrom(volumes, workspaceID))
                                 .withTerminationGracePeriodSeconds(OPENSHIFT_POD_TERMINATION_GRACE_PERIOD)
                                 .build();
```